client/views/curatorInbox/curatorSourceDetails.jade

```diff
@@ -53,9 +53,20 @@ template(name="curatorSourceDetails")
               source=source)
           if notifying
             .veil
+        else if enhancements.error
+          .curator-inbox--warning
+            i.fa.fa-warning
+            h2=enhancements.error
+            .btn.retry Retry
+        else if enhancements.processingStartedAt
```

Comment:
Is there a reason we show the notification when the document is still processing rather than continuing to show the animated loading indicator?